src/lib.rs

```diff
@@ -565,4 +637,70 @@ mod tests {
         ],
         eval: { 3.0 => 155.324961718789 },
     }
+
+    test_beat! {
+        name: less_than,
+        text: "t 64 <",
+        code: [Var, Num(64.0), Lt],
+        eval: {
+            0.0 => 1.0,
+            64.0 => 0.0,
+            128.0 => 0.0,
+        }
+    }
+
+    test_beat! {
+        name: greater_than,
+        text: "t 64 >",
+        code: [Var, Num(64.0), Gt],
+        eval: {
+            0.0 => 0.0,
+            64.0 => 0.0,
+            128.0 => 1.0,
+        }
+    }
+
+    test_beat! {
+        name: less_than_or_equal,
+        text: "t 64 <=",
+        code: [Var, Num(64.0), Leq],
+        eval: {
+            0.0 => 1.0,
+            64.0 => 1.0,
+            128.0 => 0.0,
+        }
+    }
+
+    test_beat! {
+        name: greater_than_or_equal,
+        text: "t 64 >=",
+        code: [Var, Num(64.0), Geq],
+        eval: {
+            0.0 => 0.0,
+            64.0 => 1.0,
+            128.0 => 1.0,
+        }
+    }
+
+    test_beat! {
+        name: equal_to,
+        text: "t 64 ==",
+        code: [Var, Num(64.0), Eq],
+        eval: {
+            0.0 => 0.0,
+            64.0 => 1.0,
+            128.0 => 0.0,
+        }
+    }
+
+    test_beat! {
+        name: not_equal_to,
+        text: "t 64 !=",
+        code: [Var, Num(64.0), Neq],
+        eval: {
+            0.0 => 1.0,
+            64.0 => 0.0,
+            128.0 => 1.0,
+        }
+    }
```

Comment:
We should probably put the operator tests above the example tests, not at the very bottom.